generators/java/sdk/src/main/java/com/fern/java/client/generators/endpoint/AbstractEndpointWriter.java

```diff
@@ -1689,6 +1689,26 @@ public Boolean _visitUnknown(Object unknownType) {
         }
     }
 
+    public static Optional<CodeBlock> maybeAcceptsHeader(HttpEndpoint httpEndpoint, boolean withSemiColon) {
+        String ending = withSemiColon ? ";\n" : "\n";
+
+        Set<String> contentTypes = new HashSet<>();
+
+        // TODO: We'll need to get error content types from the IR once they're available.
```

Comment:
Discussed offline - we can actually model this in a way where the `httpEndpoint` has an `accept` property that represents the set of `Accept` header values that should be sent in the request. Same idea that you're capturing here, just slightly different implementation.